package.json

```diff
@@ -64,7 +64,7 @@
     "@babel/preset-react": "^7.13.13",
     "@babel/preset-typescript": "^7.13.0",
     "@babel/runtime": "^7.14.0",
-    "@openmrs/esm-framework": "next",
+    "@openmrs/esm-framework": "^3.2.1-pre.1155",
```

Comment:
When working on the real repo we ask everyone to set these tags back to "next" after doing the `yarn upgrade`

Reply:
Okay, I will do that.